screenboards.go

```diff
@@ -9,16 +9,61 @@
 package datadog
 
 import (
+	"encoding/json"
 	"fmt"
 )
 
+// WidthS ...
+type WidthS string
```

Comment:
Why not using `PrecisionT` from https://github.com/zorkian/go-datadog-api/pull/194?

Reply:
#194 isn't merged yet.
And using `PrecisionT` as `width` or `height` several difficulty for user to consider its meaning.

If used same struct in whole this package, i think shoud be better assign general name.

Reply:
#194 was merged, you could change the name of the custom type in this PR

Reply:
i split PR at #198 .
After merge, i rebase this pr.